### PR TITLE
Makes mori+guardian impossible (and a bugfix)

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Makes mori+guardian unusable when you have either
fuck atomization all my homies hate atomization

#### Changelog

:cl:  
tweak: Using memento mori and a guardian is now impossible
bugfix: You are now told when you attempt to create a guardian with too many points
/:cl:

### CODE REVIEW

**Comment by Mesalikepie** [comment]
This is a verrrrrry rare occurrence to happen, and very inconsistent. Also there is still a counter of sharp weapon target head. I am personally against it cause it's just so rare, so let some op stuff exsist in mining if it's this inconsistent and rare. For each round this happens there's hundreds of rounds of armblades and worse (I have only gotten this twice EVER in my hundreds of hours in mining and only once did it see use which was against you ynot). Overall due to inconsistence against this let it happen when it happens

**Comment by ynot01** [comment]
tested, fully functional

**Comment by ynot01** [comment]
> This is a verrrrrry rare occurrence to happen, and very inconsistent. 
> it's just so rare, so let some op stuff exsist in mining if it's this inconsistent and rare.
> Overall due to inconsistence against this let it happen when it happens

> its literally gamebreaking but its rare so its ok

miners be like

**Comment by Mesalikepie** [comment]
I am just saying it is super inconsistent yes I am a Miner main so I am biased if it gets merged not mad. Fair enough nerf. However if I want to be broken op just go xenobio and make fucking a broodmother loyal to you. That's more consistent the Mori and guardian. I did the math also somewhat. The chance from Mori and Guardian from just 2 necro chests (I admit most experienced miners can generally get 2-4 necro chests depending on the lavaland but I haven't taken math in a year so I am keeping it easy on myself) is 1/1024 according to the wiki 32 possible necro chest loot so easy 1/32 × 1/32.

File: yogstation/code/modules/guardian/guardianbuilder.dm [hunk 1]
```diff
@@ -177,9 +177,15 @@
...
 	calc_points()
 	if(points < 0)
-		to_chat("<span class='danger'>You don't have enough points for a Guardian like that!</span>")
+		to_chat(user, "<span class='danger'>You don't have enough points for a Guardian like that!</span>")
 		used = FALSE
 		return FALSE
+	var/list/all_items = user.GetAllContents()
+	for(var/obj/I in all_items) //Check for mori
+		if(istype(I, /obj/item/clothing/neck/necklace/memento_mori))
+			to_chat(user, "<span class='danger'>The memento mori revolts at the sight of the guardian creator!</span>")
```
Comment (SomeguyManperson) [inline yogstation/code/modules/guardian/guardianbuilder.dm]: [I] revolts at the sight of the [src]!

Comment (ynot01) [reply yogstation/code/modules/guardian/guardianbuilder.dm]: u rite

**Comment by maxion12345** [comment]
Guardian and Mori is pretty broken.

Now....fix mindswap

**Comment by Hopekz** [comment]
Posting here as a reference as it might get lost on the discord (might be fixing it already)

image

**Review by fluffe9911 — APPROVED** [review]
fair and balanced

**Pull request merged**